packages/client/src/views/GrantDetails.vue

```diff
@@ -1,6 +1,10 @@
 <!-- eslint-disable max-len -->
 <template>
   <section class="container-fluid grants-details-container">
+    <b-breadcrumb>
+        <b-breadcrumb-item href="/#/grants">Home</b-breadcrumb-item>
```

Comment:
We stopped using `#` in our urls r[ecently](https://github.com/usdigitalresponse/usdr-gost/pull/2342) so I think this should just be `href="/grants"`. (Note that `/#/grants` does redirect to `/grants`, so what you have here does work, but it's kind of roundabout).

Reply:
thanks! pushed that change to the branch